### PR TITLE
rework to be inline with current ADEPT flow

### PR DESCRIPTION
**Pull request by aazaff**
@iross  How do you feel about this a rework fo the instructions?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,96 +1,58 @@
...
-- https://github.com/mclapham/app-template - Mines the literature for ichnofossil occurences 
-- https://github.com/ItoErika/Reservoir_app - Extracts geologic units acting as aquifers
-- https://github.com/bserna-usgs/app-template - Recognizes sentences related to the impact of dam removal
-- https://github.com/aazaff/usgs_geochron - Mines for geochronology measurements
-- https://github.com/adamancer/speciminer - Finds occurrences of USNM speciments mentioned in the literature
-- https://github.com/throughput-ec/UnacquiredSites/ - Extracts lat/long coordinates from sentences parsed by CoreNLP
+1. **TEXT BASED SEARCHES** The xDD development team has created the **A**utomated **D**ata **E**xtraction **P**la**T**form ([ADEPT](https://xdd.wisc.edu/adept)) to facilitate this process. The ADEPT front-end allows users to browse available documents in the xDD library using full-text search terms (powered by [ElasticSearch](elastic.co)) and other common serach parameters (e.g., publication date, journal name). Alternativey, users may intract with the xDD API (https://xdd.wisc.edu/api/v1) directly to search for relevant texts.
```
Comment (iross) [inline README.md]: Link broken.

File: README.md [hunk 2]
```diff
@@ -1,96 +1,58 @@
...
+# Building your application
 
-Open an issue here or email us at contact at geodeepdive.org to initiate the conversation.
+## Obtaining Sample Data for development purposes
+Once a user has identified a set of target publications for potential analysis, using either the [ADEPT browser](https://xdd.wisc.edu/adept#loginPanel), [xDD REST API](https://xdd.wisc.edu/api/v1), the [COSMOS system](https://github.com/UW-COSMOS/cosmos), or some combination thereof, users can request a subset of these publications for local development purposes. 
 
-## Defining an input 
-In order to deploy an application against the xDD, the _type_ and _set definition_ of input data must be defined. Fundamental questions of each are:
+Requests for development datasets can be made through the [ADEPT browser](https://xdd.wisc.edu/adept) or by contacting the xDD administrative team directly by email (https://contact@geodeepdive.org). Users will then be provided a project and user-specific URL to 200 documents randomly sampled from the identified pool of documents. This link will be provided either by email or through the ADEPT browser depending on how the sample data was requested.
```
Comment (iross) [inline README.md]: Email shouldn't have "https://"

File: README.md [hunk 3]
```diff
@@ -1,96 +1,58 @@
...
+Output must be well-defined and meet [Terms of Service](#determining-if-xdd-is-right-for-your-project). We strongly recommend that users with any doubts about the permissability of output format reach out to the xDD team directly (contact@geodeepdive.org). *Only* items found within the `/output/` directory upon application completion will be returned to users.
 
-Because xDD's corpus is large and multi-disciplinary, it is often important to restrict the set of documents used as input. This can be accomplished in several ways:
+## Creating and testing your docker image
+UW-Madison's Center for High Throughput Compouting provides a guide for creating a docker image [here](http://chtc.cs.wisc.edu/docker-build.shtml). Full documentation for the Dockerfile specification can be found [here](https://docs.docker.com/engine/reference/builder/). CHTC also provides a guide for testing a docker image locally [here](http://chtc.cs.wisc.edu/docker-test.shtml). 
 
-1. Specifying a list of article DOIs.
-2. Supplying a list of journals to target.
-3. Filtering the corpus by keyword, phrases, publication date ranges, etc.
```
Comment (iross) [inline README.md:51]: I think it's important to communicate that these kinds of set definitions are possible. Just sending them to ADEPT or the API endpoint doesn't inform potential users what handles exist to define a dataset.

**Review by iross — COMMENTED** [review]
See inline comments. I spotted a few broken links and I think we need to communicate what kind of handles a user has to define a set before we send them off to a tool that they need to learn to use.

File: README.md (hunk 1 above)
Comment (aazaff) [reply README.md]: Fxied

File: README.md (hunk 3 above)
Comment (aazaff) [reply README.md:51]: Added this back in under obtaining sample data section.

**Pull request merged**